blobfuse/blobfuse.cpp

```diff
@@ -1140,95 +1176,93 @@ read_and_set_arguments(int argc, char *argv[], struct fuse_args *args)
         std::string max_retry(cmd_options.max_retry);
         config_options.maxTryCount = stoi(max_retry);
     }
-    
+
+    if (config_options.entryTimeout == -1){
+        config_options.entryTimeout = 240;
```

Comment:
skip it from json, don't default, same reason as above

Reply:
the json doesn't get these default values so nothing to worry

Reply:
default set to 0 now